### DISCUSSION TITLE
How not to add the property "prunedAt" in .modules.yaml?

### DISCUSSION THREAD

**Discussion by tnucera** [pnpm/pnpm#4772] [Q&A] [answered]
Hello,

In my CI, i only cache the pnpm store to do "pnpm install" in next jobs. So the node_modules dir is never the same because of the changing property (a date) "prunedAt":
```
diff --color -r node_modules/.modules.yaml node_modules_/.modules.yaml
4561c4561
< prunedAt: Fri, 20 May 2022 12:58:12 GMT
---
> prunedAt: Fri, 20 May 2022 12:54:51 GMT
```
It's problematic to use Docker build cache for example.
In the meantime I delete the file property with `sed -i '/^prunedAt:/d' node_modules/.modules.yaml`.

Thx

**Comment by zkochan** [comment, marked as answer]
There is no way to not add it currently.
